cognite/neat/_session/_base.py

```diff
@@ -92,6 +102,15 @@ def verify(self) -> IssueList:
         """
         Verify the Data Model schema before the model can be written to CDF. If verification was unsuccessful, use
         `.inspect.issues()` to see what went wrong.
+
+        Example:
+
+            # From an active NeatSession
+            ...
+            >>>neat.read.csv("your_csv_file.csv", type="example", primary_key="example_id")
+            >>>neat.infer()
+            >>>neat.verify()
+            'No issues found'
```

Comment:
I use this example format when there is an output message for a successful command. In this case "No issues found"